vcfa/resource_vcfa_vcenter.go

```diff
@@ -382,3 +381,31 @@ func autoTrustHostCertificate(urlSchemaFieldName, trustSchemaFieldName string) s
 		return nil
 	}
 }
+
+func runWithRetry(runOperation func() error, retryIfErrContains string, duration time.Duration) error {
```

Comment:
Changing vCenter refresh operation approach to perform a retry if the error message contains `is currently busy`